src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -29,122 +29,140 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/
   - /docs/introduction-new-relic-ai
   - /docs/alerts/new-relic-alerts/getting-started/alerting-new-relic
+  - /docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/
 signupBanner:
   text: Monitor and improve your entire stack. 100GB free. Forever.
 ---
 
-import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.webp'
-
-As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening, where it's happening, and what to do about it. Alerts allow you to define the conditions in your system you want to know about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that proactively notifies your team about potential problems by identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues, and providing a root cause analysis. 
-
-New Relic's alerts and applied intelligence were designed for busy DevOps teams to end response fatigue: you can detect and diagnose incidents and prioritize them for immediate action. 
+import accountsAlertingConcepts from 'images/accounts_diagram_alerting-concepts.webp'
+
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed and New Relic's alerts and applied intelligence are the answer. With alerts, you can configure, detect, and diagnose incidents, prioritize them, and take the needed immediate action. Use alerts to identify anomalies before they reach your end-user. 
```

Comment:
I think this could be a good place to use the framework that Ally used [here ](https://build-dc14add3-3e69-410b-9d3d-7460e1b3a39b.gatsbyjs.io/docs/new-relic-fleet-manager)and Rob used [here](https://docs.newrelic.com/docs/change-tracking/change-tracking-introduction/). After reading this intro I am still a little confused about what alerts do vs. what applied intelligence does and what you can do with both features. (I know that's not your fault because it's a very confusing set of features!) But perhaps you could break these points into their own bullets:

Reply:
- Define your conditions
- Use alert conditions
- Maybe mentions something about anomalies? 
- alert coverage gaps?

Reply:
Like build upon a story:

You define what an issue is, you notify your team when there's an issue, and then you let New Relic help you identify issues you didn't even know were there.

Reply:
Hey @homelessbirds, thank you, I've modified the intro after reading these comments!